pages/common/ffmpeg.md

```diff
@@ -4,16 +4,20 @@
 
 - Extract the sound from a video and save it as MP3:
 
-`ffmpeg -i {{video_filename}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound.mp3}}`
+`ffmpeg -i {{path/to/video}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound}}.mp3`
```

Comment:
Is the filename required to me `.mp3`?

Reply:
The example mentions "save it as MP3". This is to make that clearer. And since its in `common/`, so sometimes Windows barfs if it doesn't see `.mp3` extension.

Reply:
Ahhh ok. Cool :smiley_cat: